packages/exa.rb

```diff
@@ -0,0 +1,36 @@
+require 'package'
+
+class Exa < Package
+  description 'Modern replacement for \'ls\''
+  homepage 'https://the.exa.website'
+  version '0.9.0'
+  source_url 'https://github.com/ogham/exa/archive/v0.9.0.tar.gz'
+  source_sha256 '96e743ffac0512a278de9ca3277183536ee8b691a46ff200ec27e28108fef783'
+
+  depends_on 'cmake'
+  depends_on 'rust'
```

Comment:
You don't need the `depends_on 'cmake'` line since that is already included in "core".  Also, since rust is only a build dependency, you don't need to require it if installing from binary: `depends_on 'rust' => :build`.